src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -247,6 +247,12 @@ public static function collectDeprecations($outputFile)
             }
             $deprecations[] = array(error_reporting(), $msg);
         });
+        // This can registered before the PHPUnit error handler.
```

Comment:
typo: missing "be" (... can be...

Reply:
Fixed in latest push - I did a rebase to get the latest changes so had to force push.